src/general/OptionsMenu.tscn

```diff
@@ -373,6 +376,72 @@ min_value = 10.0
 max_value = 50.0
 value = 20.0
 
+[node name="HBoxContainer6" type="HBoxContainer" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer"]
+margin_top = 261.0
+margin_right = 648.0
+margin_bottom = 283.0
+
+[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer6"]
+margin_right = 42.0
+margin_bottom = 22.0
+text = "GPU:"
+
+[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer6"]
+margin_left = 46.0
+margin_right = 564.0
+margin_bottom = 22.0
+size_flags_horizontal = 3
+
+[node name="GpuInfo" type="Label" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Graphics/VBoxContainer/HBoxContainer6"]
+margin_left = 568.0
+margin_right = 648.0
+margin_bottom = 22.0
+text = "GPU Info"
```

Comment:
In addition to the translation documentation linked which also this covered: for texts that will be overridden in the code, `PLACEHOLDER` should be added to the editor description so this placeholder text would not be fetched by the translation system.